### DISCUSSION TITLE
3.0.0-beta08

### DISCUSSION THREAD

**Discussion by Him188** [open-ani/animeko#130] [Announcements]
- 重新设计视频数据源选择
   - 现在可以直观地查看完整原标题, 来源, 发布时间等信息, 即使 Ani 有时候匹配不准标题也能人工选择
      > 以前只会显示 "xx 字幕组 1", "xx 字幕组 2"
   - 优化了自动选择的逻辑, 现在只会在加载完成列表之后自动选择一次
      > 以前会额外重复选择一次, 导致视频重新加载 (这样也导致了开播延迟)
   - 请求会在失败时重试, 提升稳定性
   - 现在支持按上次选择的字幕语言等信息精准匹配下一集的资源
      > 按条目记忆. 每个条目只需要第一集选择一次, 后面就会自动选择
- 支持 Mikan 数据源
- 搜索资源时同时使用中文翻译和原文
   - 解决了有的新番搜不到的问题, 现在都能都到了
- 加快默认弹幕速度
   > 现在是上个版本的两倍速度
- 全屏时将视频居中
   > 强迫症福音
- 调整图标色彩

<img width="400" src="https://github.com/Him188/ani/assets/12100985/927cd5e2-dd18-48cd-84ce-061efcd67910">

<img width="400" src="https://github.com/Him188/ani/assets/12100985/23f6f57d-5b6e-450e-9d99-836dc019ee6e">

----

**附件:**

- `.zip` 是包含 JRE 的解压即用的绿色版本；
- `.dmg`、`.deb`、`.rpm` 和 `.msi` 分别是 macOS、Debian、RedHat 和 Windows 平台的安装程序；
- `.exe` 是包含 JRE 的 Windows 程序，可以直接运行。
- `.jar` 是不包含 JRE 的最小程序，需要配合 JRE 11 及以上使用；
- `.apk` 是 Android 应用，支持的最低 Android 版本为 8.0；

注意，桌面平台仅支持 64 位操作系统。在使用 macOS 的安装程序时可能提示文件损坏，这是因为此版本应用没有签名，可以忽略错误继续使用。

也可扫描以下二维码下载 Android 版本：
![](https://github.com/Him188/ani/releases/download/v3.0.0-beta08/ani-3.0.0-beta08.apk.qrcode.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/Him188/ani/releases/tag/v3.0.0-beta08'>3.0.0-beta08</a>.</em>

**Comment by NeKoOuO** [comment]
現在的版本有辦法處理那種一到十二的那種合輯嗎

**Reply by Him188** [reply]
不能，会只能播放第一个文件

这个要增加一层，比较麻烦的，一步一步来
